From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@jfoshee__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@jfoshee Thanks for the contribution. I'll send this to the author for review.

alancameronwills commented:
Thanks Jacob!
Dene, please pull.

rothja committed:
Updating information about how the portal configures storage

rothja committed:
Fixing validation error

carolinacmoravia committed:
CC42709: Update-AzureRMVmss incorrect parameter fix #53

Hello, @iainfoulds,

This proposed file change comes from this Community Contribution: https://github.com/MicrosoftDocs/azure-docs.nl-nl/pull/53/files
Please, help to check this proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

dlepow committed:
N-series: updates for NCv2 and ND

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into GPUv2

dlepow committed:
Updates draft complete

dlepow committed:
format fix

dlepow committed:
Added quota note

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into GPUv2

dlepow committed:
link fix

daveba committed:
initial draft of application manifest rewrite

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into GPUv2

dlepow committed:
Feedback from Tariq

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into GPUv2

dlepow committed:
Updated main sizes page with NCv2 and ND

daveba committed:
more updates

daveba committed:
"playing with table formatting"

daveba committed:
fixed table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

daveba committed:
table formatting

jlian committed:
Added new instructions for change subscription directory

jlian committed:
Retired old edit directory article since it's not billing scenario

sethmanheim committed:
Updates for GA

sethmanheim committed:
Fixes after reviewing

sethmanheim committed:
Trying to fix link warnings

jlian committed:
Acrolinx and TOC fixes

daveba committed:
table formatting

sethmanheim committed:
One more try

daveba committed:
table formatting

jlian committed:
Fix redirect syntax

sethmanheim committed:
Wording tweaks

jlian committed:
Clean up screenshot

jlian committed:
Add cursor to screenshot

jlian committed:
Fix another typo in redirect

Commit:
Added Russian, Norwegian, Danish.

rothja committed:
Updating FAQ with new structure and VM lifecycle

kgremban committed:
New troubleshooting topic for Edge

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geodr

rothja committed:
Updating FAQ

rothja committed:
Fixing link

Commit:
draft of sentiment topic

Commit:
draft of sentiment topic minor edits

rothja committed:
Fixing links

rothja committed:
Fixing formatting and adding link to SOS

daveba committed:
changing author value in this commit-next commit will delete

daveba committed:
changing author value in this commit-next commit will delete

daveba committed:
duplicate article deletion

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dec12-troubleshoot

kgremban committed:
Added TOC entry and removed numbered steps

GitHubber17 committed:
edit pass: documentation-government-connect-ssms

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dec12-troubleshoot

GitHubber17 committed:
firewall rule capitalization

Commit:
draft of sentiment topic - major edits

Commit:
draft of sentiment topic - formatting edits

daveba committed:
updating based on feedback

daveba committed:
updating based on feedback

daveba committed:
updating based on feedback

daveba committed:
updating based on feedback

rjagiewich committed:
edit pass: active-directory-saas-cisco-webex-tutorial.md

rjagiewich committed:
edit pass: active-directory-saas-cisco-webex-tutorial.md

daveba committed:
updating based on feedback

rjagiewich committed:
edit pass: active-directory-saas-cisco-webex-tutorial.md

daveba committed:
updating based on feedback

Commit:
fixed link

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geodr

daveba committed:
updating based on feedback

Commit:
final edit pass

kevinvngo committed:
Update refresh date

kevinvngo committed:
UserVoice link

Commit:
toc edit for new topic

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-r-content-updates

kevinvngo committed:
Removed redundancy and updated refresh date

kevinvngo committed:
ADLS data source

kevinvngo committed:
removed migration utilitiy

ShawnJackson committed:
Update documentation-government-connect-ssms.md

daveba committed:
adding redirects

dlepow committed:
Restored missing Linux NV content

dlepow committed:
LIS and CUDA version updates from Josh

kevinvngo committed:
Updated Portal Images - Support ticket

daveba committed:
deleting article

dlepow committed:
Updated CUDA to 9.1 and release numbers

rothja committed:
Updating overview with performance section

kevinvngo committed:
Add Monitor Ticket

kevinvngo committed:
Refactor support ticket  updated images

kevinvngo committed:
nit change to Monitor a support ticket

rjagiewich committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-saas-cisco-webex-tutorial.md

rjagiewich committed:
edit pass: active-directory-saas-cisco-webex-tutorial

raiye committed:
Update cloud-services-guestos-msrc-releases.md

dlepow committed:
Acrolinx

kevinvngo committed:
Update sql-data-warehouse-get-started-create-support-ticket.md

kevinvngo committed:
Addressed non-blocking issues

kevinvngo committed:
Grammar - Non blocking issue

daveba committed:
updating based on feedback

GitHubber17 committed:
edit pass: adjustments

rjagiewich committed:
Update active-directory-saas-cisco-webex-tutorial.md

v-viinde committed:
USER STORY 1144946 SaaS App Tutorial: OneTrust Privacy Management Software

kevinvngo committed:
Grammar and nit changes

Commit:
Added Configure guest OS for multiple NICs section

@iainfoulds Please sign-off if you're OK with the change. The addition of this section brings this article to parity with the Linux version of this article. https://docs.microsoft.com/en-us/azure/virtual-machines/linux/multiple-nics#configure-guest-os-for-multiple-nics.

sethmanheim committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into geodr

sethmanheim committed:
Adding samples links

sethmanheim committed:
One more link

barbkess committed:
Add SQL DW to the list of databases

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-r-content-updates

twooley committed:
new b2b article and TOC update

ShawnJackson committed:
Merge pull request #29978 from GitHubber17/doc-gov-conn-sms

edit pass: documentation-government-connect-ssms

Commit:
Article and TOC updates per team feedback

ShawnJackson committed:
Update active-directory-saas-cisco-webex-tutorial.md

GitHubber17 committed:
Merge pull request #28630 from dlepow/GPUv2

N-series: updates for NCv2 and ND

daveba committed:
fixed based on feedback

PRMerger12 committed:
Merge pull request #26787 from rothja/perfupdate5

Updating information about how the portal configures storage

GitHubber17 committed:
Merge pull request #30089 from kevinvngo/patch-18

Troubleshoot doc, migration utility, remove DWU9000/18000

tynevi committed:
Merge pull request #30134 from barbkess/hub

Add SQL DW to the list of databases

rothja committed:
Fixing spelling and formatting errors

raiye committed:
Update cloud-services-guestos-msrc-releases.md

ShawnJackson committed:
Merge pull request #30003 from rjagiewich/active-directory-saas-cisco-webex-tutorial.md

Edit pass: Active directory saas cisco webex tutorial.md

Commit:
review edits

miaojiang committed:
Update api-management-transformation-policies.md

PRMerger17 committed:
Merge pull request #30097 from raiye/patch-2

Update cloud-services-guestos-msrc-releases.md

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dec12-troubleshoot

neilpeterson committed:
updated link

Commit:
Updates for planned maintenance

twooley committed:
new b2b article and TOC update

mahesh-unnikrishnan committed:
Update active-directory-ds-admin-guide-configure-secure-ldap-enable-ldaps.md

iainfoulds committed:
Freshness updates and revisions

daveba committed:
redirecting zen article to top level saas tutorial list

kgremban committed:
Emmanuel's feedback

Commit:
Merge pull request #29871 from LuisCabrer/patch-11

Added Russian, Norwegian, Danish.

twooley committed:
new b2b article and TOC update

PRMerger14 committed:
Merge pull request #30144 from neilpeterson/draft-link-fix

updated link

kgremban committed:
formatting

Commit:
fixed link

Commit:
cloud services clarification

PRMerger10 committed:
Merge pull request #30143 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-admin-guide-configure-secure-ldap-enable-l…

GitHubber17 committed:
Merge pull request #29257 from MicrosoftDocs/active-directory-application-manifest

initial draft of application manifest rewrite

Commit:
quickfix08

mmacy committed:
[aci] availability update

Commit:
added another redirect to fix 404

Commit:
capitalization

PRMerger18 committed:
Merge pull request #30148 from mmacy/aci-availability

[aci] availability update

GitHubber17 committed:
Merge pull request #30139 from jeffgilb/jeff1215

Article and TOC updates per team feedback

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-r-content-updates

sethmanheim committed:
Caps in H1

iainfoulds committed:
Updates to MongoDB doc

nitinme committed:
Adding an include

Lisaco88 committed:
Merge pull request #30136 from twooley/twb2bhybrid3

new b2b article and TOC update

Commit:
Merge pull request #30149 from MarkusVi/quickfix08

quickfix08

iainfoulds committed:
Updates to docker-machine

GitHubber17 committed:
Merge pull request #29926 from kgremban/dec12-troubleshoot

New troubleshooting topic for Edge

daveba committed:
fixing redirect syntax error

nitinme committed:
another update

Lisaco88 committed:
Merge pull request #29845 from sethmanheim/geodr

Updates for GA

kgremban committed:
Fixing small typos in troubleshooting doc

Commit:
link and acrolinx

PRMerger17 committed:
Merge pull request #30154 from nitinme/nitinme-r-content-updates

Addding include to overview articles

MikeDodaro committed:
In Video Search landing page: images -> videos

iainfoulds committed:
Updates to unmanaged disk conversion

PRMerger19 committed:
Merge pull request #30127 from jimdial/patch-46

Added Configure guest OS for multiple NICs section

kgremban committed:
Update troubleshoot.md

PRMerger14 committed:
Merge pull request #28544 from carolinacmoravia/patch-478

CC42709: Update-AzureRMVmss incorrect parameter fix #53

PRMerger17 committed:
Merge pull request #30141 from miaojiang/patch-6

Update api-management-transformation-policies.md

Lisaco88 committed:
Merge pull request #29952 from MicrosoftDocs/saas-app-delete-duplicate-articles

Saas app delete duplicate articles

GitHubber17 committed:
Merge pull request #29908 from rothja/vmpolicy

Updating FAQ with new structure and VM lifecycle

PRMerger12 committed:
Merge pull request #30155 from kgremban/dec12-troubleshoot

Fixing small typos in troubleshooting doc

tfitzmac committed:
changed example templates

Commit:
Joerg fixes

iainfoulds committed:
Freshness checks and updates

rjagiewich committed:
Merge pull request #30117 from v-viinde/onetrust

USER STORY 1144946 SaaS App Tutorial: OneTrust Privacy Management Sof…

danieldobalian committed:
fix spacing bug

Lisaco88 committed:
Merge pull request #29834 from jlian/edit-directory-ibiza

Edit directory in ibiza

Lisaco88 committed:
Merge pull request #29939 from bradsev/tdsp-sentiment-analysis-sample

draft of sentiment topic

iainfoulds committed:
Updating MongoDB doc

PRMerger13 committed:
Merge pull request #30165 from danieldobalian/ca-fix

fix spacing bug

PRMerger17 committed:
Merge pull request #30164 from iainfoulds/vmfreshness121517-batch2

VM freshness updates

Lisaco88 committed:
Merge pull request #30150 from georgewallace/omslink

oms fix : added another redirect to fix 404

rjagiewich committed:
Merge pull request #30145 from iainfoulds/freshness121517-batch1

VM freshness updates

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #30157 from MikeDodaro/videoLandingPageTypoFix

In Video Search landing page: images -> videos

Lisaco88 committed:
Merge pull request #30161 from tfitzmac/1215multiple

changed example templates

v-alje committed:
Merge pull request #30170 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #30142 from cynthn/ziv-update

Updates for planned maintenance
